packages/commonwealth/client/scripts/views/components/CommunityStake/useCommunityStake.ts

```diff
@@ -68,7 +68,7 @@ const useCommunityStake = (props: UseCommunityStakeProps = {}) => {
     stakeData?.vote_weight,
   );
   const stakeBalance = Number(userStakeBalanceData);
-  const stakeValue = stakeBalance * Number(buyPriceData?.price);
```

Comment:
Assuming this was intentionally changed to reflect stakeValue is the price instead of price * account balance

Reply:
The buyPriceData is the price of the total stakeBalance(ie how much to buy my stake balance) so it doesnt need the multiplier. In the future I think we'll change this to sell to represent sell side vwap but just wanted something that works for now(found the bug in existing code)